modules/storage-persistent-storage-efs-storage-class.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-efs.adoc
+
+[id="efs-storage-class_{context}"]
+= Create the EFS StorageClass
+
+Before persistent volume claims can be created, a StorageClass
+must exist in the {product-title} cluster. The following instructions
+create the StorageClass for the EFS provisioner.
+
+.Procedure
+
+. Define an {product-title} ConfigMap containing the environment
+variables by creating a `storageclass.yaml` with the following contents:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: aws-efs
+provisioner: openshift.org/aws-efs
+parameters:
+  gidMin: "2048" <1>
+  gidMax: "2147483647" <2>
+  gidAllocate: "true" <3>
+----
+<1> An optional argument that defines the minimum GID for volume
+assignments. Defaults to `2048`.
+<2> An optional argument that defines the maximum GID for volume
+assignments. Defaults to `2147483647`.
+<3> An optional argument that determines if GIDs are assigned to
+volumes. If `false`, dynamically provisioned volumes will not be
+allocated GIDs, allowing all users to read and write to the
+created volumes. Defaults to `true`.
+
+. Once the file has been configured, it may be created in your cluster
```

Comment:
s/Once/After

! once adv
Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”

Do you need to do this to proceed? If so, s/it may be created/create it